test/benchmark/nginx/http_req10k_conc1_bw/run.sh

```diff
@@ -6,5 +6,6 @@ set -e
 
 cp /benchmark/nginx/nginx.conf /usr/local/nginx/conf/
 
+echo "All preparations are done. Starting the benchmark..."
```

Comment:
This sentence has been repeated triple times, which is bad. And it is inobvious - at least in this file - that this is a special message that must be kept in sync with the one in `host_guest_bench_runner.sh`.

Reply:
I replace it with "BENCHMARK_VM_READY_FOR_TEST"  now.

Reply:
I find a more simple way. Just print the ready message "The VM is ready for the benchmark" like a normal banner in the `bench_runner.sh` and then it will be displayed in all benchmarks.